packages/material-ui-lab/src/TreeItem/TreeItem.test.js

```diff
@@ -85,6 +88,9 @@ describe('<TreeItem />', () => {
     expect(getIcon('6'))
       .attribute('data-test')
       .to.equal('endIcon');
+    expect(getIcon('7'))
```

Comment:
I think a better text would either be to look for a group or the aria attribute.

Reply:
is this ok?

Reply:
The test looks okay but can you move it out of the icon test please? It should be fine as a top-level test with the description something like: "should treat an empty array equally to no children".

Reply:
done